diff --git a/node_modules/expo-symbols/build/SymbolView.js b/node_modules/expo-symbols/build/SymbolView.js
index 51239d4..9fe6f52 100644
--- a/node_modules/expo-symbols/build/SymbolView.js
+++ b/node_modules/expo-symbols/build/SymbolView.js
@@ -1,4 +1,4 @@
-import { useFonts } from '@expo-google-fonts/material-symbols';
+import { useFonts } from '@expo-google-fonts/material-symbols/useFonts';
 import { useMemo } from 'react';
 import { Platform, PlatformColor, Text, View } from 'react-native';
 import { androidSymbolToString } from './android';
diff --git a/node_modules/expo-symbols/src/SymbolView.tsx b/node_modules/expo-symbols/src/SymbolView.tsx
index a30a73a..171aff7 100644
--- a/node_modules/expo-symbols/src/SymbolView.tsx
+++ b/node_modules/expo-symbols/src/SymbolView.tsx
@@ -1,3 +1,3 @@
-import { useFonts } from '@expo-google-fonts/material-symbols';
+import { useFonts } from '@expo-google-fonts/material-symbols/useFonts';
 import { useMemo, type JSX } from 'react';
 import { Platform, PlatformColor, Text, View } from 'react-native';
